plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestPhoenixDistributedQueries.java

```diff
@@ -126,6 +127,17 @@ public void testInsert()
         assertUpdate("DROP TABLE test_insert");
     }
 
+    @Override
+    public void testInsertArray()
+    {
+        assertThatThrownBy(super::testInsertArray)
+                // TODO (https://github.com/prestosql/presto/issues/6421) array with double null stored as array with 0
```

Comment:
is `double` significant? I got the same error with `array(BIGINT)` typed `a` column. May be we can generalize the issue and todo.

Reply:
I didn' test this thoroughly. Added a comment in the issue, but otherwise kept the comment message intact, as it is not incorrect and describes the test's problem precisely.